src/QsFmt/Formatter/Printer.fs

```diff
@@ -5,15 +5,14 @@ module Microsoft.Quantum.QsFmt.Formatter.Printer
 
 open Microsoft.Quantum.QsFmt.Formatter.SyntaxTree
 open Microsoft.Quantum.QsFmt.Formatter.SyntaxTree.Trivia
-open System
 
 /// <summary>
 /// Prints a <see cref="Trivia"/> node to a string.
 /// </summary>
 let printTrivia =
     function
     | Whitespace ws -> ws
-    | NewLine -> Environment.NewLine
+    | NewLine nl -> nl
```

Comment:
I think should keep this as Environment.NewLine to keep this flexible with different local setups. My local setup, for example is a mix of different new line characters, and having this be a specific new line, one way or the other, breaks most of my tests. My recommendation is to leave this as Environment.NewLine and instead use a function like the following on inputs before string-comparison:
```
/// <summary>
/// Replaces New Line characters in source string for Environment.NewLine characters.
/// </summary>
let private standardizeNewLines (source: string) =
    source.Replace("\r", "").Replace("\n", Environment.NewLine)
```